starter-code/scripts/models/repo.js

```diff
@@ -4,12 +4,21 @@
   var reposObj = {};
 
   reposObj.allRepos = [];
-// TODO: create a githubToken.js file that we can use to generate our headers
-         // properly.
+  // TODO:Done? create a githubToken.js file that we can use to generate our headers
+  // properly.
+
   reposObj.requestRepos = function(callback) {
     /* TODO: How would you like to fetch your repos? Someone say AJAX?!
-      Do not forget to call the callback! */
+    Do not forget to call the callback! */
+    $.ajax({
+      url: 'https://api.github.com/user/repos?type=owner',
+      method: 'GET',
+      headers: {
+        Authorization: 'token ' + githubToken
+      }
+    });
```

Comment:
You should be doing something upon a successful response from Github (allowing the repos contained within the response to be accessed later).